shaarli_client/client/v1.py

```diff
@@ -304,3 +314,7 @@ def delete_tag(self, resource, params):
         """Delete a tag"""
         self._check_endpoint_params('delete-tag', params)
         return self._request('DELETE', 'tags/%s' % resource, params)
+
+    def delete_link(self, resource):
+        """Delete a link"""
+        return self._request('DELETE', 'links/%d' % resource, {})
```

Comment:
you need to add `, verify_certs` as the last argument to `_request()` - https://github.com/shaarli/python-shaarli-client/blob/master/shaarli_client/client/v1.py#L249

Reply:
None of the others functions have the `verify_certs` argument

Reply:
I will recheck everything, `verify_certs` was added in January but tests have been failing for longer than that (https://github.com/shaarli/python-shaarli-client/pull/46). It's possible I missed someting